libosmscout-map/src/osmscoutmap/oss/Parser.cpp

```diff
@@ -1662,7 +1662,6 @@ void Parser::AREABORDERSYMBOLSTYLE(StyleFilter filter, bool state) {
 
 void Parser::ROUTESTYLE(StyleFilter filter, bool state) {
 		LinePartialStyle style;
-		std::string      slot;
```

Comment:
I am worry that this file is generated by Coco generator. This change will be overwritten by next update style upate. It seems that support for "slot" of routes is not implemented yet...